Vienna/Sources/Fetching/RefreshManager.m

```diff
@@ -99,7 +99,7 @@ -(instancetype)init
         config.HTTPAdditionalHeaders = @{@"User-Agent": userAgent};
         config.HTTPMaximumConnectionsPerHost = 6;
         config.HTTPShouldUsePipelining = YES;
-        _urlSession = [NSURLSession sessionWithConfiguration:config delegate:self delegateQueue:[NSOperationQueue mainQueue]];
+        _urlSession = [NSURLSession sessionWithConfiguration:config delegate:self delegateQueue:nil];
```

Comment:
Is this not going to cause any problems with the delegate callbacks? The delegate queue would probably a private background queue.

Reply:
> The queue should be a serial queue, in order to ensure the correct ordering of callbacks. If `nil`, the session creates a serial operation queue for performing all delegate method calls and completion handler calls.

So this change should not affect the sequence of delegate callbacks.